lib/appsignal/integrations/code_ownership.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Appsignal
+  module Integrations
+    # @!visibility private
+    module CodeOwnershipIntegration
+      class << self
+        def before_complete(transaction, error)
+          team = nil
+          begin
+            team = ::CodeOwnership.for_backtrace(error.backtrace)
+          rescue => ex
+            logger = Appsignal.internal_logger
+            logger.error(
+              "Error while looking up CodeOwnership team: #{ex.class}: #{ex.message}\n" \
+                "#{ex.backtrace.join("\n")}"
+            )
+          end
+
+          return if team.nil?
+
+          transaction.add_tags(:owner => team.name)
```

Comment:
Can it be a falsy value that is valid? Like it returns `false` for some reason and we want to set `false`?

If not, this may be better:

```suggestion
          team = ::CodeOwnership.for_backtrace(error.backtrace)
          transaction.add_tags(:owner => team.name)
        rescue => ex
          logger = Appsignal.internal_logger
          logger.error(
            "Error while looking up CodeOwnership team: #{ex.class}: #{ex.message}\n" \
              "#{ex.backtrace.join("\n")}"
          )
```